c2rust-analyze/src/main.rs

```diff
@@ -493,6 +495,30 @@ fn run(tcx: TyCtxt) {
         info.lasn.set(lasn);
     }
 
+    // For testing, putting #[c2rust_analyze_test::fixed_signature] on a function makes all
+    // pointers in its signature FIXED.
+    for &ldid in &all_fn_ldids {
+        if !util::has_test_attr(tcx, ldid, TestAttr::FixedSignature) {
+            continue;
+        }
+        let lsig = match gacx.fn_sigs.get(&ldid.to_def_id()) {
+            Some(x) => x,
+            None => continue,
+        };
```

Comment:
@spernsteiner, I vaguely remember you answering this before, but I can't find the comment.  Why do we `continue` here instead of panicking?  When would the lookup fail?

Reply:
Not sure.  I tried changing the `continue` to a `panic!()` just now and nothing broke in the tests (though only a few of those use `fixed_signature`).